### DISCUSSION TITLE
Details about RT-dependent normalization method

### DISCUSSION THREAD

**Discussion by SummerJuneLee** [vdemichev/DiaNN#1062] [Q&A]
Dear Vadim,
I hope this message finds you well. I am currently using DIA-NN and came across some questions regarding its implementation. After reviewing information from the Nature and discussions on GitHub, I would appreciate clarification on the detailed mechanism of RT-dependent normalization.

Although Nature paper and GitHub page explained about global normalization that top 3 precursors with least CVs are picked and the sum of those precursors' intensities are normalized to have the same value(PLEASE TELL ME if my explanation is wrong), it didn't contain information about RT-dependent normalization mechanism.

To introduce you how far I approached this matter, all I found out about RT-dependent normalization is that 

1. RT-dependent normalization is recommended

2. It is used to reduce the loss of hydrophilic or hydrophobic peptides in specific samples

Furthermore, I found this discussion explaining that "extra normalisation factor is applied calculated as the running median of the measured fold changes (sample vs other samples) across the retention time, which corrects for effects such as loss of hydrophilic or hydrophobic peptides in specific sample compared to other samples." (https://github.com/vdemichev/DiaNN/discussions/955)

I am having trouble grasping the concept fully, which is why I have additional questions. 

1. How is the fold change calculated when comparing the sample of interest to multiple other samples? Is it simply calculated as the ratio of the sample intensity to the average intensity of the other samples?
2. What parameter is used to calculate the fold change? Is it derived from the sum of intensities of all peptides at specific retention times? A detailed explanation would be greatly appreciated.
3. Why is RT-dependent normalization recommended, not global normalization?

Thank you for your attention to this matter.

Warm regards, 
Summer
Graduate Student

**Comment by vdemichev** [comment]
Hi Summer,

The details of the algorithm are not published yet.

1. It's like calculating a separate normalisation factor for each RT value, using just peptides eluting in the vicinity of this RT.
3. It happens quite often that specific runs have relative depletion of hydrophobic or hydrophilic peptides, or just peptides around a specific RT. RT-dependent normalisation corrects for it. Further, it's in general more robust than global normalisation (algorithm is a bit different and better), hence it's always recommended.

Best,
Vadim

**Reply by SummerJuneLee** [reply]
Thank you :)

**Reply by SummerJuneLee** [reply]
Dear Vadim, I have one more question. Can I use additional normalization(i.e. Quantile normaliztion) even after RT-dependent normalization, or is it unrecommended?

**Reply by vdemichev** [reply]
Yes, can do! In particular, can make sense to apply extra normalisation on the protein level

**Reply by SummerJuneLee** [reply]
Thank you very much. Hope you have a great day today!

**Reply by omicsbisai** [reply]
> Yes, can do! In particular, can make sense to apply extra normalisation on the protein level

Dear Vadim, refer to this answer, I want to know whether the extra normalisation on the protein level is too much for the dataset. Will the differences bewteen the samples be compressed?

Best,
omicsbisai

**Reply by vdemichev** [reply]
Hi omicsbisai,

If done correctly, extra protein-level normalisation can only be beneficial, or will just have no effect.
No, the 'true differences' will not be compressed.

Best,
Vadim

**Reply by omicsbisai** [reply]
> Hi omicsbisai,
> 
> If done correctly, extra protein-level normalisation can only be beneficial, or will just have no effect. No, the 'true differences' will not be compressed.
> 
> Best, Vadim

Thanks a lot!

**Reply by omicsbisai** [reply]
> Hi omicsbisai,
> 
> If done correctly, extra protein-level normalisation can only be beneficial, or will just have no effect. No, the 'true differences' will not be compressed.
> 
> Best, Vadim

Hi,
I wanna know if the extra peptide-level normalisation needed. Casue I know the DIANN did this on the precursors level.
Thanks a lot.
Best,
omicsbisai

**Reply by vdemichev** [reply]
I don’t think extra peptide-level normalisation can help
